gtk4/src/gesture_stylus.rs

```diff
@@ -0,0 +1,25 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::GestureStylus;
+use gdk::AxisUse;
+use glib::translate::*;
+
+impl GestureStylus {
+    #[doc(alias = "gtk_gesture_stylus_get_axes")]
+    pub fn get_axes(&self, axes: Vec<AxisUse>) -> Option<Vec<f64>> {
+        let mut values: Vec<f64> = Vec::new();
+        unsafe {
+            let mut axes1: Vec<gdk::ffi::GdkAxisUse> = axes.iter().map(|a| a.to_glib()).collect();
+            axes1.push(gdk::ffi::GDK_AXIS_IGNORE);
+            if from_glib(ffi::gtk_gesture_stylus_get_axes(
+                self.to_glib_none().0,
+                axes1.as_mut_ptr(),
+                values.as_mut_ptr() as *mut *mut f64,
```

Comment:
This seems wrong. AFAIU this function returns an allocated array of doubles here. So this would be something like

```rust
let mut values = ptr::null_mut();
..._get_axes(..., &mut values);
```

And then you have `axes.len()` values in there.